fluent-plugin-google-cloud.gemspec

```diff
@@ -22,7 +22,7 @@ eos
   gem.add_runtime_dependency 'fluentd', '~> 0.10'
   gem.add_runtime_dependency 'googleapis-common-protos', '~> 1.3'
   gem.add_runtime_dependency 'google-api-client', '~> 0.9.0'
-  gem.add_runtime_dependency 'google-cloud-logging', '~> 0.23.2'
+  gem.add_runtime_dependency 'google-cloud-logging', '~> 1.2.1'
```

Comment:
This actually only needs to be upgraded to 0.24.2 as the commit that added the "trace" LogEntry field is https://github.com/GoogleCloudPlatform/google-cloud-ruby/commit/f9fb517bdfa8dedfcb477f9188546d62f5c85e6a.

That means we would be taking in changes incorporated from 12/17/16 to 3/4/17 to the gem. Here's a relevant link in case it helps review those changes: https://github.com/GoogleCloudPlatform/google-cloud-ruby/commits/master?before=0a277ce77bb1c0a4a3eeceddd7f1ab3615740dca+70&path%5B%5D=google-cloud-logging